package.json

```diff
@@ -24,16 +24,25 @@
     "type-check": "tsc",
     "test": "jest",
     "test:env": "LOAD_ENV=1 yarn test",
+    "test:clean": "find src -name __recordings__ -prune -exec rm -rf {} \\;",
     "test:ci": "yarn lint && yarn type-check && yarn test",
     "build": "tsc -p tsconfig.dist.json --declaration && cp README.md dist/README.md && cp -r jupiterone/ dist/jupiterone/",
     "prepush": "yarn lint && yarn type-check && jest --changedSince main"
   },
   "peerDependencies": {
-    "@jupiterone/integration-sdk-core": "^7.4.1"
+    "@jupiterone/integration-sdk-core": "^8.1.3"
   },
   "devDependencies": {
-    "@jupiterone/integration-sdk-core": "^7.4.1",
-    "@jupiterone/integration-sdk-dev-tools": "^7.4.1",
-    "@jupiterone/integration-sdk-testing": "^7.4.1"
+    "@jupiterone/integration-sdk-core": "^8.1.3",
+    "@jupiterone/integration-sdk-dev-tools": "^8.1.3",
+    "@jupiterone/integration-sdk-testing": "^8.1.3",
+    "@types/node": "^17.0.8",
+    "@types/node-fetch": "^2.5.12",
+    "@types/uuid": "^8.3.3"
+  },
+  "dependencies": {
+    "@types/got": "^9.6.12",
+    "got": "^11.8.2",
```

Comment:
I know in other integrations we use [node-fetch](https://github.com/node-fetch/node-fetch), but I am personally a fan of [got](https://github.com/sindresorhus/got) and opted to use it here. It has retry support, hooks in the request lifecycle and other features. In the event we have to code around the intermittent API errors mimecast currently has, its creature comforts should serve us well.

Reply:
I've asked about standardizing to a library (also have several using axios that I know of... and have written 😅).  I don't know that we've decided as a team to standardize on one yet, so it's good to know the rationale for this choice.

Reply:
let the request-library wars begin! 😉

Reply:
Once we *do* standardize on one, I'd like to update the template to have it listed instead of `import http from 'http';` to nudge us in that direction for future graph- projects